### ISSUE TITLE
Add support for attr-bans-regex?

### ISSUE THREAD

**Issue by pdehaan** [htmllint/htmllint#236]
Currently we have a "need" to try and ban any "on*" handlers in our code. For example,

```html
<a href="#" onclick="blah()">...</a>
```

Is there a way to block all "on*" global style event handlers without needing to explicitly define all ~48?
https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers#Properties_2

**Comment by pdehaan** [comment]
Regex may also be nice, because if I recall correctly, the `attr-bans` are case sensitive, so `onclick=""` and `onClick=""` were subtly different.

**Comment by mlochbaum** [comment]
Just released v0.8.0. This version allows a regular expression for `attr-bans`, and also fixes all the case sensitivity issues I could find. Attribute and tag names should be matched case insensitively.
